containers/proxy-squid-image/uyuni-configure.py

```diff
@@ -1,22 +1,31 @@
 #!/usr/bin/python3
+# pylint: disable=invalid-name
+"""Configure script for Uyuni proxy squid container."""
 
 import os
 import re
 import yaml
 
 # read from file
-with open("/etc/uyuni/config.yaml") as source:
+with open("/etc/uyuni/config.yaml", encoding="utf-8") as source:
     config = yaml.safe_load(source)
 
     # read to existing config file
-    with open("/etc/squid/squid.conf", "r+") as config_file:
+    with open("/etc/squid/squid.conf", "r+", encoding="utf-8") as config_file:
         file_content = config_file.read()
-        file_content = re.sub(r"cache_dir aufs .*", f"cache_dir aufs /var/cache/squid {str(config['max_cache_size_mb'])} 16 256", file_content)
-        file_content = re.sub(r"access_log .*", "access_log stdio:/proc/self/fd/1 squid", file_content)
+        file_content = re.sub(
+            r"cache_dir aufs .*",
+            f"cache_dir aufs /var/cache/squid {str(config['max_cache_size_mb'])} 16 256",
+            file_content,
+        )
+        file_content = re.sub(
+            r"access_log .*", "access_log stdio:/proc/self/fd/1 squid", file_content
+        )
+        file_content += "\n" + "include /etc/squid/conf.d/*" + "\n"
```

Comment:
if we are always adding this line, why not to add it to the squid configuration directly?

Reply:
It has sense, but I guess the same is valid also for `access_log` param, am I right  ?
cc @cbosdo

Reply:
The `access_log` would mess up the non-container install, so it has to stay here until we completely remove this installation method for Uyuni.

Reply:
right, so the same is valid also for this `include`. I'll keep the change here